openapi/index.yaml

```diff
@@ -99,15 +145,15 @@ definitions:
         type: string
         x-extensible-enum:
           - REVOKED
-      revokation_date:
+      revocation_date:
         $ref: "#/definitions/Timestamp"
-      motivation:
+      reason:
         type: string
         minLength: 1
     required:
       - status
-      - revokation_date
-      - motivation
+      - revocation_date
+      - reason
```

Comment:
since we have `revocation_date` probably it's better to have `revocation_reason`

Reply:
Ok, I'll rename it :)

Reply:
Fixed in [2a43cf6](https://github.com/pagopa/io-functions-cgn/pull/10/commits/2a43cf6f61f802ab9d4fd4b018c7e9176eed8865)